### ISSUE TITLE
IOSXR -- Improper parsing for "show bgp instance all all all summary"

### ISSUE THREAD

**Issue by kokasha** [CiscoTestAutomation/genieparser#92]
The Current Parser Only capture the following Address Families
- IPv4 Unicast
- VPNv4 Unicast
- IPv6 Unicast
- VPNv6 Unicast

However, the parser doesn't capture the following address families

- IPv4 Labeled-unicast
- IPv6 Labeled-unicast
- l2vpn vpls-vpws
- IPv4 multicast
- IPv6 multicast

Can these Address Families be added to the parser

**Comment by xiaoxinz-cisco** [comment]
Hello @kokasha ,

Thank you for reporting.

About the first three:
- IPv4 Labeled-unicast
- IPv6 Labeled-unicast
- l2vpn vpls-vpws
Current regular expression does not capture those.
Current regex ([where it is in code](https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxr/show_bgp.py#L4505)):
```
p3 = re.compile(r'^\s*Address *Family:'
                        ' *(?P<address_family>[a-zA-Z0-9\s]+)$')
```

To fix it:
```
p3 = re.compile(r'^\s*Address *Family:'
                        ' *(?P<address_family>[\S\s]+)$')
```

About these two:
- IPv4 multicast
- IPv6 multicast 
They should be captured if they are also in the format:
``` Address Family: IPv4 multicast```

Generally, we release once per month, to save your waiting time,
I recommend you could fix this little problem on your own and create a pull request. 
Our team will review and merge them.

We appreciate your understanding and contribution. 😃 

Thanks,
Irene

**Comment by xiaoxinz-cisco** [comment]
Hi @kokasha ,

Our support team will take over this issue and will add it in the next release.

Thanks,
Irene

**Comment by xiaoxinz-cisco** [comment]
Hello @kokasha ,

We fixed the issue and genie package **20.2.1b1** has been released with the fix.
Could you please run the following command and try again to see if it works?

**pip install genie.libs.parser --upgrade --pre**

Please try and let's know if there is an issue.

Thanks,
Irene
